lsp-diagnostics.el

```diff
@@ -115,12 +115,12 @@ g. `error', `warning') and list of LSP TAGS."
                (bitmap (or (get flycheck-level 'flycheck-fringe-bitmaps)
                            (get flycheck-level 'flycheck-fringe-bitmap-double-arrow))))
           (flycheck-define-error-level new-level
```

Comment:
lets format that with flycheck loaded

Reply:
If i add `(require 'flycheck)` on top of file. Can I?

Reply:
flycheck is an optional dependency but at the same time, most of the devs work with flycheck already loaded.

Reply:
ok, I will add `flycheck` as a cask `dev-dependency`.